app/src/main/java/io/homeassistant/companion/android/settings/developer/DeveloperSettingsFragment.kt

```diff
@@ -93,14 +110,24 @@ class DeveloperSettingsFragment : DeveloperSettingsView, PreferenceFragmentCompa
     }
 
     override fun onThreadDebugResult(result: String, success: Boolean?) {
-        threadDebugDialog?.hide()
+        activeTaskDialog?.hide()
         AlertDialog.Builder(requireContext())
             .setTitle(commonR.string.thread_debug)
             .setMessage("${if (success == true) "✅" else if (success == null) "⚠️" else "⛔"}\n\n$result")
             .setPositiveButton(commonR.string.ok, null)
             .show()
     }
 
+    override fun onWebViewClearCacheResult(success: Boolean) {
+        lifecycleScope.launch {
+            delay(750L) // Add a delay to prevent the dialog 'flashing' if clearing completed quickly
+            activeTaskDialog?.hide()
+            if (success) {
+                Toast.makeText(requireContext(), commonR.string.clear_webview_cache_success, Toast.LENGTH_SHORT).show()
+            }
```

Comment:
If it fails the user won't know anything about it, and will report in the issue that he did clear the cache, that might be misleading.

Reply:
Added a message on failure. This one is a bit theoretical so there's not really anything specific to add to the message, I've yet to see clearing cache fail.